tsl/src/continuous_aggs/insert.c

```diff
@@ -181,20 +181,31 @@ cache_inval_entry_init(ContinuousAggsCacheInvalEntry *cache_entry, int32 hyperta
 }
 
 static inline void
-cache_entry_switch_to_chunk(ContinuousAggsCacheInvalEntry *cache_entry, Oid chunk_id,
+cache_entry_switch_to_chunk(ContinuousAggsCacheInvalEntry *cache_entry, Oid chunk_reloid,
 							Relation chunk_relation)
 {
-	Chunk *modified_tuple_chunk = ts_chunk_get_by_relid(chunk_id, false);
+	Chunk *modified_tuple_chunk = ts_chunk_get_by_relid(chunk_reloid, false);
 	if (modified_tuple_chunk == NULL)
-		elog(ERROR, "continuous agg trigger function must be called on hypertable chunks only");
+	{
+		ereport(ERROR,
+				(errcode(ERRCODE_INTERNAL_ERROR),
+				 errmsg("continuous agg trigger function must be called on hypertable chunks only"),
+				 errdetail("Called on '%s'.", get_rel_name(chunk_reloid))));
+	}
 
 	cache_entry->previous_chunk_relid = modified_tuple_chunk->table_id;
 	cache_entry->previous_chunk_open_dimension =
 		get_attnum(chunk_relation->rd_id,
 				   NameStr(cache_entry->hypertable_open_dimension.fd.column_name));
 
 	if (cache_entry->previous_chunk_open_dimension == InvalidAttrNumber)
-		elog(ERROR, "continuous agg trigger function must be called on hypertable chunks only");
+	{
+		ereport(ERROR,
+				(errcode(ERRCODE_INTERNAL_ERROR),
+				 errmsg("open dimension '%s' not found in chunk %s",
+						get_rel_name(chunk_relation->rd_id),
+						NameStr(cache_entry->hypertable_open_dimension.fd.column_name))));
```

Comment:
```suggestion
						NameStr(cache_entry->hypertable_open_dimension.fd.column_name),
						get_rel_name(chunk_relation->rd_id))));
```
Chunk and column names was inverted!

Reply:
Oops, thank you!